### ISSUE TITLE
effected by JVM bug JDK-8212155?

### ISSUE THREAD

**Issue by eknv** [krzysztofslusarski/continuous-async-profiler#27]
For JVM versions prior to 17, the async-profiler is affected by the following JVM related bug:
https://github.com/async-profiler/async-profiler/issues/154
https://bugs.openjdk.org/browse/JDK-8212155

Is continuous-async-profiler also effected by this issue?

**Comment by krzysztofslusarski** [comment]
Yes. continuous-async-profiler is affected by all the issues that affect async-profiler
